haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -47,9 +47,13 @@ BUILD()
 		cp $LIB ../lib/$NAME
 	done
 	cd ../
-	cd EventEditor && make OBJ_DIR=objects && cd ../
-	cd EventServer && make OBJ_DIR=objects && cd ../
-	cd PreferencesPreflet && make OBJ_DIR=objects
+	make -C EventEditor OBJ_DIR=objects
+	make -C EventServer OBJ_DIR=objects
+	make -C PreferencesPreflet OBJ_DIR=objects
+	
+	#cd EventEditor && make OBJ_DIR=objects && cd ../
+	#cd EventServer && make OBJ_DIR=objects && cd ../
+	#cd PreferencesPreflet && make OBJ_DIR=objects
```

Comment:
Don't forget to remove these 3 lines.

BTW, I've got a small variant:
```
	make OBJ_DIR=objects -C EventEditor
	make OBJ_DIR=objects -C EventServer
	make OBJ_DIR=objects -C PreferencesPreflet
```
But if you prefer to have the `-C` before the `OBJ_DIR=objects`, thats OK for me.

Reply:
Okay :)